foia_hub/views.py

```diff
@@ -67,16 +77,15 @@ def get_agency_list():
 ###
 
 def learn(request):
-    return HttpResponse(env.get_template('learn.html').render(request=request))
+    return render(request, 'learn.html', {'request': request})
```

Comment:
If you add the request preprocessor, you can replace these all with generic views, just passing in the template name w/in the urls.py file.